src/components/toolbox/autoSuggest/index.js

```diff
@@ -29,6 +31,9 @@ class AutoSuggest extends React.Component {
     this.onSelectItem = this.onSelectItem.bind(this);
     this.resetListIndex = this.resetListIndex.bind(this);
     this.handleUpdateIndex = this.handleUpdateIndex.bind(this);
+    this.onBlur = this.onBlur.bind(this);
+    this.onFocus = this.onFocus.bind(this);
+    this.isOnError = this.isOnError.bind(this);
```

Comment:
If you use arrow function, you don't need these.

Reply:
hey Ali thanks for the review totally agree with you, I like more to use arrow functions in class instead this, however we did't use that standard so far.

I think you are talking about this.

```
class Name extends Component {

  onBlur = () => {
    ...
  }

  render() {
    return (
      <Component onBlur={this.onBlur} />
    )
  }
}
```

If you are talking of use the arrow functions in the render directly we try to not use unless be really necesary because cause performance issues, not that the app wont work but it is not a good practice.

Reply:
I agree that arrow functions are less code than binding everything, but it terms of consistency, we use `bind` in all class-based components. So if we want to set a new standard for how to handle this, I'd say the solution should be to use react Hooks and no class-based components.

We already agreed on using hooks in the last technical meeting.